packages/node/src/ethereum/api.service.ethereum.test.ts

```diff
@@ -83,4 +84,16 @@ describe('ApiService', () => {
     console.log('Finalized height', height);
     expect(height).toBeGreaterThan(16_000_000);
   });
+
+  it('ensure api errorCode is exposed when throwing', async () => {
+    let thrownError: CustomError;
+    try {
+      await apiService
+        .safeApi(17520376)
+        .getCode('0x75F0398549C9fDEa03BbDde388361827cb376D5');
+    } catch (e) {
+      thrownError = e;
+    }
+    expect(thrownError.code).toBe('INVALID_ARGUMENT');
```

Comment:
This isn't perfect but its the pattern for expecting exceptions from promises

```suggestion
    expect(() => apiService
        .safeApi(17520376)
        .getCode('0x75F0398549C9fDEa03BbDde388361827cb376D5')
    ).rejects.toThrow('INVALID_ARGUMENT');
```